app/shared/productTypes.tsx

```diff
@@ -219,28 +236,55 @@ export const ProductTypes: { [productKey: string]: ProductType } = {
     urlPart: "paper",
     validator: (me: MeResponse) => me.contentAccess.paperSubscriber,
     includeGuardianInTitles: true,
-    alternateReturnToAccountDestination: domainSpecificSubsManageURL
+    alternateManagementUrl: domainSpecificSubsManageURL,
+    alternateManagementCtaLabel: (productDetail: ProductDetail) =>
+      productDetail.tier === "Newspaper Delivery"
+        ? "To manage your holiday stops"
+        : undefined,
+    productPage: "subscriptions"
   },
   guardianweekly: {
     friendlyName: "Guardian Weekly",
     allProductsProductTypeFilterString: "Weekly",
     urlPart: "guardianweekly",
-    validator: (me: MeResponse) => true, // TODO: change to me.contentAccess.weeklySubscriber once exposed by members-data-api
-    alternateReturnToAccountDestination: domainSpecificSubsManageURL
+    validator: (me: MeResponse) => me.contentAccess.weeklySubscriber,
+    alternateManagementUrl: domainSpecificSubsManageURL,
+    alternateManagementCtaLabel: () => "To renew your Guardian Weekly",
+    productPage: "subscriptions"
   },
   digipack: {
     friendlyName: "digital pack",
     allProductsProductTypeFilterString: "Digipack",
     urlPart: "digitalpack",
     validator: (me: MeResponse) => me.contentAccess.digitalPack,
+    showTrialRemainingIfApplicable: true,
+    productPage: "subscriptions"
+  },
+  soCalledSubscriptions: {
+    friendlyName: "subscription",
+    allProductsProductTypeFilterString: "SoCalledSubscription",
+    urlPart: "subscriptions",
+    validator: (me: MeResponse) =>
+      me.contentAccess.digitalPack ||
+      me.contentAccess.paperSubscriber ||
+      me.contentAccess.weeklySubscriber,
     productPage: {
-      title: "Digital Pack",
-      navLink: navLinks.digitalPack,
+      title: "Subscriptions",
+      navLink: navLinks.subscriptions,
       noProductInTabCopy:
         "To manage your existing membership or contribution, please select from the tabs above.",
       tierRowLabel: "Subscription product",
-      showSubscriberId: true,
-      showTrialRemainingIfApplicable: true
+      showSubscriptionId: true
+    },
+    mapSoCalledToSpecific: (productDetail: ProductDetail) => {
```

Comment:
We are matching on strings, and we are only expecting one of three strings, so it feels like we could use a type to help us out here?

```
type subscription = "DigitalPack" | "Newspaper" | "Digital Pack"
```

As an aside, we typically use tier in the context of membership only, so I'm slightly surprised to find these values under productDetail.tier.

Reply:
also the types aren't available at **runtime**, so string literal types are only useful when passing known strings around